flutter_google_places_sdk/android/src/main/kotlin/com/msh/flutter_google_places_sdk/FlutterGooglePlacesSdkPlugin.kt

```diff
@@ -90,17 +92,36 @@ class FlutterGooglePlacesSdkPlugin : FlutterPlugin, MethodCallHandler {
       requestBuilder.setOrigin(originLocation)
     }
 
+    call.argument<Map<String, Map<String, Double?>?>>("bounds")?.let { bounds ->
+      var northeast = bounds["northeast"] ?: mapOf()
+      var southwest = bounds["southwest"] ?: mapOf()
+
+      var locationBias = RectangularBounds.newInstance(
+        LatLng(
+          southwest["lat"] ?: 0.0, 
+          southwest["lng"] ?: 0.0
+        ),
+        LatLng(
+          northeast["lat"] ?: 0.0, 
+          northeast["lng"] ?: 0.0
+        )
+      )
```

Comment:
Not sure if setting the default value location bias to (0,0,0,0) will help us. If the value is not provided I think the best option is just to not set the location bias

Reply:
Then shouldn't we do the same to the origin?

Reply:
Yeah, you're right. Maybe the best option would be to raise an exception, but considering we're already checking for the key existence before, I think it's safe to use a default value in these cases 👍🏼